scripts-python/xmlquerry

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+
+# Authors: Jim Edwards , Andreas Wilke
+
+from standard_script_setup import *
+
+from CIME.utils import expect, run_cmd
+from CIME.case import Case
+
+import argparse, doctest, shutil, glob
+import xml.etree.ElementTree as ET
+import textwrap
+
+
+###############################################################################
+def parse_command_line(args, description):
+###############################################################################
+
+    # my %opts = (
+    #     fileonly        =>0,
+    #     valonly        =>0,
+    #     value            =>0,
+    #     file =>0,
+    #     noexpandxml            =>0,
+    #     help        =>0,
+    #     listall        =>0,
+    #     loglevel        =>"INFO",
+    #     );
+
+
+    
+    epilog="""\
```

Comment:
Please put this in a docstring at the top of the file. You can then refer to this string with __doc__. You can see create_test as an example.